services/celery/celery.py

```diff
@@ -1,16 +1,129 @@
-# from components import OrderManager
-# from services import DydxWithdrawal
-# from settings_config.celery import app
-#
-#
-# @app.task(name="check_withdrawal", default_retry_delay=4 * 60)
-# def check_withdrawal():
-#     dydx_withdrawal = DydxWithdrawal()
-#     order_manager = OrderManager()
-#     result = dydx_withdrawal.all_transfer_details(
-#         {"limit": 100, "transfer_type": "WITHDRAWAL"}
-#     )
-#     transfers = result["data"]["transfers"]
-#     for transfer in transfers:
-#         if transfer["status"] == "CONFIRMED":
-#             order_manager.update_data(transfer["id"], "withdrawal", "CONFIRMED")
+import time
+from components import OrderManager
+from services import DydxWithdrawal, LoadContracts, DydxOrder, DydxAdmin
+from tests.constants import SEVEN_DAYS_S
+from settings_config.celery_config import app
+
+open_position = False
+order_id = "5d90fda25c94f91a85a3b8bf3081a09be442e00941ed11c5b31c6dc890cc464"
+
+
+@app.task(name="check_withdrawal", default_retry_delay=4 * 60)
+def check_withdrawal():
+    dydx_withdrawal = DydxWithdrawal()
+    order_manager = OrderManager()
+    result = dydx_withdrawal.all_transfer_details(
+        {"limit": 100, "transfer_type": "WITHDRAWAL"}
+    )
+    transfers = result["data"]["transfers"]
+    for transfer in transfers:
+        if transfer["status"] == "CONFIRMED":
+            order_manager.update_data(transfer["id"], "withdrawal", "CONFIRMED")
+
+
+@app.task(name="open_order_on_dydx", track_started=True)
+def open_order_on_dydx():
+    global open_position
+    global order_id
+
+    if open_position is False:
+        print(open_position)
+        load_contract = LoadContracts()
+        market_price = load_contract.get_asset_price(
+            "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419"
+        )  # make it dynamic
+        print(market_price)
+        trigger_price = 1750
```

Comment:
can add a comment here #TODO: set formula for trigger_price so that we don't forget changing this in the future